templates/kdump_conf

```diff
@@ -0,0 +1,3 @@
+path /{{inventory_hostname}}
```

Comment:
If the target name for this file is `kdump.conf` then it should be named `kdump.conf` here, as well instead of `kdump_conf`. Otherwise, you lose any potential for an editor to properly grok the syntax highlighting.